### DISCUSSION TITLE
summary/2024/

### DISCUSSION THREAD

**Discussion by giscus** [Shitao5/shitao-blog#38] [Comments]
# summary/2024/

工作：
[&hellip;] 生活：
[&hellip;] 下周：
[&hellip;] 工作：
[&hellip;] 生活：
[&hellip;] 下周：
[&hellip;] 工作：
[&hellip;] 生活：
[&hellip;] 下周：
[&hellip;] 工作：
[&hellip;] 生活：
[&hellip;] 下周：
[&hellip;] 工作：
[&hellip;] 生活： &hellip;

https://shitao5.org/summary/2024/

**Comment by LFG12** [comment]
祝早日康复

**Comment by LFG12** [comment]
涛哥您好，想问下您总结页上的运动情况是如何实现的呀，方便给个链接吗
还想请问一下这个运动情况是要每天自己输入还是读取软件的数据呀
万分感激，祝您生活愉快涛哥

**Reply by Shitao5** [reply]
你好呀，叫我诗涛就可以噢嘿嘿。这个总结页的运动情况是每周手动录入[运动数据](https://github.com/Shitao5/shitao-blog/blob/main/content/summary/2024/sport.csv)，然后在网站生成的时候利用 R 语言自动[绘制](https://github.com/Shitao5/shitao-blog/blob/main/content/summary/2024/index.Rmd#L11-L43)的。
